### ISSUE TITLE
Unable to find variable COLORS

### ISSUE THREAD

**Issue by ziotom78** [ihalila/pancurses#11]
The documentation often refers to the variable `COLORS`, which however seems to be defined nowere:

`````
$ grep "\<COLORS\>" $(find . -type f)
./src/lib.rs:/// must be a legal color value, i.e., 0 through COLORS-1, inclusive. The values that are returned
./src/lib.rs:/// inclusive. The foreground and background must be between 0 and `COLORS` - 1, inclusive. If the
./src/lib.rs:/// of COLORS according to this function -- 16 for FALSE, and 8 for TRUE.
./src/lib.rs:/// and white), and two global variables; `COLORS` and `COLOR_PAIRS` (respectively defining the
`````

**Comment by gyscos** [comment]
Same for `COLOR_PAIRS`.

**Comment by ihalila** [comment]
COLORS() and COLOR_PAIRS() are now available in version 0.9
